packages/core-contracts/contracts/proxy/ProxyStorage.sol

```diff
@@ -9,8 +9,8 @@ contract ProxyStorage {
 
     function _proxyStorage() internal pure returns (ProxyNamespace storage store) {
         assembly {
-            // bytes32(uint(keccak256("io.synthetix.proxy")) - 1)
-            store.slot := 0x9dbde58b6f7305fccdc5abd7ea1096e84de3f9ee47d83d8c3efc3e5557ac9c74
+            // bytes32(uint(keccak256("io.synthetix.v3.core-contracts.proxy")) - 1)
```

Comment:
np. Do you prefer just proxy or v3.core-contracts.proxy?

Reply:
Just proxy for now! But yes, this decision is to be finalized before going to prod.